components/AudioPlayer.module.css

```
@@ -0,0 +1,198 @@
+.audio_player_section_container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  padding: 10%;
+}
+
+.image_wrapper {
+  position: relative;
+  top: 30px;
+  width: 180px;
+  height: 180px;
```

```suggestion
  /* we can adjust the position with a space using margin or padding */
  width: 180px;
  height: 180px;
```
@bluwakpa I noticed that you tend to use `top` to adjust each element's position.
I personally don't really see a lot to adjust the position this way.  It's usually used margin or padding.
Also, try not to use **absolute units** and try to use **relative units** such as `xx rem` or `yy %` to make it easier to control the style in a variety of devices.

https://www.tutorialrepublic.com/css-tutorial/css-units.php#:~:text=The%20units%20in%20which%20length,be%20treated%20as%20an%20error.